### ISSUE TITLE
Change default setting for How to resolve quiz attempts, thanks

### ISSUE THREAD

**Issue by Urpokarhu1** [jpahullo/moodle-tool_mergeusers#186]
Hello,

Please change default setting "How to resolve quiz attempts tool_mergeusers | quizattemptsaction" from "Default: Do nothing: do not merge nor delete" to "merge attempts from both users and renumber" I think this would be a good thing to do by default and not notice it after you have merged 200 accounts.

**Comment by mifantrax** [comment]
This would be very welcome, we just had a case where we discovered the surprise of all quiz attempts lost, could there be a parameter to choose what happens with quiz attempts? Or maybe it is there but we cannot see? Thank you so much!

**Comment by jpahullo** [comment]
Hi!

Thanks for the feedback.

We'll consider your words @Urpokarhu1 . If you can, you could provide a PR with that change, This is the easiest way to get things evolving.

Regarding to your question, @mifantrax , if you merged user A into B, with option "Do nothing" on quizzes, you can update the setting to "merge attemps from both users and renumber" and repeat mergin user A into B. This would have the same effect in the end. The second merging will only merge missing operations, and, in particular, attempts were not deleted from user A.

Thanks for your time!

**Comment by christianabila** [comment]
Another way to prevent such mistakes is to replace the global setting `tool_mergeusers | quizattemptsaction` with the same setting but on user merge level.  In other words, this setting has to be set each time a user merge needs to be done (in mergeusers/index.php).

In fact, to go even further, when merging users, a new step could be introduced between the current steps "1. Choose users to merge'" and "2. Confirm users to merge", e.g. "2. Merge strategy", where the current global "General settings"

- `tool_mergeusers | suspenduser`
- `tool_mergeusers | quizattemptsaction`
- `tool_mergeusers | uniquekeynewidtomaintain`

are moved to and where they need to be set.

And in the already existing step "2. Confirm users to merge", which would become "3. Confirm users to merge", the selected values of the settings can be reviewed alongside the selected users.

I think this would be a good option.

In summary, it is easy to forget that the global setting `tool_mergeusers | quizattemptsaction`  exists if user merges happen only occasionally. Needing them to be set at each merge user action can prevent unwanted loss of data.

**Comment by jpahullo** [comment]
Thanks @christianabila  for your feedback, 

It could be interesting in general, but not for our usecase in our institution.

So, I encourage you to provide a patch in the form of a Pull Request that implements such a confirmation step on the web merge. CLI merges will still use the default plugin settings, since they can be implemented in a non-interactive fashion.

Thanks for your support,

Jordi
